pkg/resource/container_image.go

```diff
@@ -173,12 +173,12 @@ func (r *ContainerImage) cloneRemoteResource(ctx context.Context, path string, r
 			for i, m := range indexManifest.Manifests {
 				if m.Platform.Architecture == r.options.Arch {
 					index = i
-					logger.Info("selected first %s container", r.options.Arch)
+					r.Info(logger.CloneError, "selected first %s container", r.options.Arch)
```

Comment:
```suggestion
					r.Info(logger.CloneDetail, "selected first %s container", r.options.Arch)
```

Thoughts, since it's not really an error? 

I was thinking Detail over Info since it could work for Debug and other non-error logs too.